functional_widget/test/src/success.dart

```diff
@@ -18,7 +18,7 @@ Widget noArgument() => Container();
 Widget required(foo, int bar) => Container();
 
 @widget
-Widget named({foo, int bar}) => Container();
+Widget named({foo, int bar = 42}) => Container();
```

Comment:
Same thing, a test on its own instead of modifying `named`, which shouldn't have an impact on `equality_test.dart`

Reply:
Done